addons/editor-compact/addon.json

```diff
@@ -12,6 +12,12 @@
       "link": "https://scratch.mit.edu/users/Maximouse/"
     }
   ],
+  "info": [
+    {
+      "id": "hide-icons-update",
+      "text": "To hide menu bar icons, use the \"Customizable menu bar\" addon."
```

Comment:
Technically, the convention is to start the addon name with lowercase (for reference, see all the info's that reference "developer tools")